### ISSUE TITLE
MNIST Dataloader error

### ISSUE THREAD

**Issue by GitDT11** [CuthbertCai/pytorch_DANN#15]
There is a minor error in the train and test dataloaders for the MNIST dataset. The transform for normalizing data values receives mean and standard deviation for three channels whereas the MNIST images only have a single channel. We can just fix this by changing the values in the transforms such as:
```
if dataset == 'MNIST':
        transform = transforms.Compose([
                                        transforms.ToTensor(),
                                        transforms.Normalize(mean= params.dataset_mean[0], std= params.dataset_std[0])
                                        ])
```
This way only one channel is transformed as it should.
